### PR TITLE
change worker_saturation default value to 1.1 in the document

### PR DESCRIPTION
**Pull request by minhnguyenxuan60**
Closes #xxxx

- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

Admins can comment `ok to test` to allow this one PR to run or `add to allowlist` to allow all future PRs from the same author to run.

File: docs/source/scheduling-state.rst [hunk 1]
```diff
@@ -84,7 +84,7 @@ forgotten
     dereferenced from the scheduler.
 
 .. note::
-    When the ``distributed.scheduler.worker_saturation`` config value is set to ``inf``
```
Comment (fjetter) [inline docs/source/scheduling-state.rst:87]: The documentation itself is still correct, the only thing that changed is the default. If this value is set to `inf`, there is no intermediate state, etc.

However, we changed this and the default is now 1.1 (see section below).

It likely makes sense to rearrange these paragraphs to first talk about a finite value which is the default and then talk about the non-default `inf` value

Comment (minhnguyenxuan60) [reply docs/source/scheduling-state.rst:87]: I have moved the paragraph as you suggested.

File: docs/source/scheduling-state.rst [hunk 2]
```diff
@@ -84,17 +84,17 @@ forgotten
...
-    (default), there's no intermediate state between ``waiting`` / ``no-worker`` and
+    Setting ``distributed.scheduler.worker_saturation`` config value to ``1.0``
+    (default) will queue excess root tasks on the scheduler in the ``queued`` state.
```
Comment (hendrikmakait) [inline docs/source/scheduling-state.rst:88]: Please note that the default is `1.1`, not `1.0`. We should also mention that this behavior is valid for any finite value, not just the default. For example:

```suggestion
    Setting ``distributed.scheduler.worker_saturation`` config value to ``1.1``
    (default) or any other finite value will queue excess root tasks on the scheduler in the ``queued`` state. 
```

**Review by hendrikmakait — CHANGES_REQUESTED** [review]

File: docs/source/scheduling-state.rst (hunk 2 above)
Comment (minhnguyenxuan60) [reply docs/source/scheduling-state.rst]: I've changed it like what you suggested. Please tell me if there is anything else that I should do.

**Review by hendrikmakait — APPROVED** [review]
Thanks for your contribution, @minhnguyenxuan60!

**Pull request merged**